### PR TITLE
Changing to Postgres Documentation Updated

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/administer/changing-to-postgresql.md [hunk 1]
```diff
@@ -1,97 +1,147 @@
 # Changing to PostgreSQL
 
-By default, WSO2 products use the embedded H2 database as the database
+By default, WSO2 Identity Server use the embedded H2 database as the database
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: WSO2 Identity Server uses

File: en/docs/administer/changing-to-postgresql.md [hunk 2]
```diff
@@ -1,97 +1,147 @@
...
-default, `         WSO2_CARBON_DB        ` datasource is used to connect
-to the default  H2 database, which stores registry and user management
-data. After setting up the PostgreSQL database to replace the default H2
-database, either change the default configurations of the
-`          WSO2_CARBON_DB         `datasource, or configure a new datasource to point it to the new database as explained below.
+default, `WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
+to the default  H2 database. Here `WSO2_SHARED_DB` is the database which stores registry and user management
+data.`         WSO2_IDENTITY_DB        ` is a specific database for the identity server which stores
+identity related data. After setting up the PostgreSQL database. You can point the `WSO2_IDENTITY_DB` or 
+`WSO2_SHARED_DB` or both to that PostgreSQL database by following below instructions.
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: `WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
to the default  H2 database. 
-   `WSO2_SHARED_DB`: This database stores registry and user management data.
-   `WSO2_IDENTITY_DB`: This is database stores identity-related data. 

After setting up the PostgreSQL database, you can point the `WSO2_IDENTITY_DB` or `WSO2_SHARED_DB` or both to that PostgreSQL database by following below instructions.

File: en/docs/administer/changing-to-postgresql.md [hunk 3]
```diff
@@ -1,97 +1,147 @@
...
+      <td><strong>port</strong></td>
+      <td>The port of the database.</td>
+      </tr>
+      <tr class="even">
+      <td><strong>name</strong></td>
+      <td>The name of the database.</td>
+      </tr>
+      </table>   
+ 
+ Sample configuration is given below.
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: A sample configuration is given below.

File: en/docs/administer/changing-to-postgresql.md [hunk 4]
```diff
@@ -1,97 +1,147 @@
...
-    [database.shared_db.pool_options]
-    maxActive = "80"
-    maxWait = "60000"
-    minIdle = "5"
-    testOnBorrow = true
-    validationQuery="SELECT 1; COMMIT"
-    validationInterval="30000"
-    defaultAutoCommit=false
-    ```
+Apart from above basic configurations wso2 supports advanced database configurations.
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: WSO2 Identity Server

File: en/docs/administer/changing-to-postgresql.md [hunk 5]
```diff
@@ -1,97 +1,147 @@
...
+   ``` toml
+       [database.identity_db.pool_options]
+        maxActive = "80"
+        maxWait = "60000"
+        minIdle = "5"
+        testOnBorrow = true
+        validationQuery="SELECT 1; COMMIT"
+        validationInterval="30000"
+        defaultAutoCommit=false
+   ```
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: Fix alignment issues.

File: en/docs/administer/changing-to-postgresql.md [hunk 6]
```diff
@@ -1,97 +1,147 @@
...
+   ``` toml
+      [database.shared_db.pool_options]
+       maxActive = "80"
+       maxWait = "60000"
+       minIdle = "5"
+       testOnBorrow = true
+       validationQuery="SELECT 1; COMMIT"
+       validationInterval="30000"
+       defaultAutoCommit=false
+   ```
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: Fix alignment issues.

File: en/docs/administer/changing-to-postgresql.md [hunk 7]
```diff
@@ -168,127 +209,62 @@ Follow the steps below to change the type of the default datasource.
...
+                commitOnReturn="true"
+           ```
+        - `WSO2_SHARED_DB` `deployment.toml` Configurations.
+                
+           ``` toml
+              [database.shared_db.pool_options]
+               defaultAutoCommit="false"
+               commitOnReturn="true"
+           ```    
+
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: Fix alignment issues.

File: en/docs/administer/changing-to-postgresql.md [hunk 8]
```diff
@@ -168,127 +209,62 @@ Follow the steps below to change the type of the default datasource.
...
+                defaultAutoCommit="false"
+                rollbackOnReturn="true"
+           ```
+        - `WSO2_SHARED_DB` `deployment.toml` Configurations.
+                
+           ``` toml
+              [database.shared_db.pool_options]
+               defaultAutoCommit="false"
+               rollbackOnReturn="true"
+           ```
```
Comment (YvonneW) [inline en/docs/administer/changing-to-postgresql.md]: Fix alignment issues.

File: en/docs/administer/changing-to-postgresql.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 3 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 4 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 5 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 6 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 7 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

File: en/docs/administer/changing-to-postgresql.md (hunk 8 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-postgresql.md]: done

**Pull request merged**